dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/util/RuntimeGenerator.java

```diff
@@ -87,7 +96,7 @@ public static void main(String[] args) throws IOException {
                         try {
                             File testDir = new File(testOutput+name);
                             Scanner testReader = new Scanner(testDir);
-                            File temp = new File("new_csv.txt");
+                            File temp = new File("new_xml.txt");
```

Comment:
What is this file for? Can we provide a more descriptive name?

Reply:
@winglam Its a temporary file to save and later parse through sootTimerOutput data. I will rename this to more descriptive name in next pull request.